gnome-extra/nautilus-dropbox/nautilus-dropbox-2019.02.14.ebuild

```diff
@@ -0,0 +1,58 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="6"
+
+GNOME2_LA_PUNT="yes"
+PYTHON_COMPAT=( python3_{6,7,8} )
+
+inherit autotools python-single-r1 gnome2
```

Comment:
I didn't see any .schema files being installed / handled, maybe *gnome2* is unneeded here?